docs: Better documentation for unhandled rejections warnings (#1099)
getsentry/sentry-docs: src/collections/_documentation/platforms/node/index.md

All our JavaScript-related SDKs provide the same API. Still, there are some differences between them which this section of the docs explains.

{% capture __alert_content -%}

> The default configuration of the SDK captures unhandled promise rejections, which makes Node.js not emit default warnings about them. If you want to restore this behaviour, use `--unhandled-rejections=warn` flag to run your code.

Reviewer: ```suggestion
The default configuration of the SDK captures unhandled promise rejections, which prevents Node.js from emitting default warnings about them. If you want to restore this behavior, use `--unhandled-rejections=warn` flag to run your code.
```
